src/ray/common/id.cc

```diff
@@ -259,42 +203,28 @@ TaskID ObjectID::TaskId() const {
 ObjectID ObjectID::ForPut(const TaskID &task_id, ObjectIDIndexType put_index) {
```

Comment:
As you mentioned in the description I think it'd be best to collapse these into one definition for clarify - something like `ObjectID::FromIndex`

Reply:
Definitely. What do you think about `FromTaskIndex`? I feel like that pretty clearly conveys the semantics, not sure if including `Task` is redundant.

Reply:
Preference for `FromIndex` - `Task` is redundant and this can either be a `PutIndex` or a `ReturnIndex`